espressostreamer/espresso_streamer.go

```diff
@@ -52,9 +52,13 @@ func NewEspressoStreamer(namespace uint64, hotshotUrls []string,
 	pollingHotshotPollingInterval time.Duration,
 	parentChainNodeUrl string,
 	headerReaderConfig headerreader.Config,
-	espressoTEEVerifierAddress common.Address,
+	espressoTEEVerifierAddress string,
 ) *EspressoStreamer {
 
+	if !common.IsHexAddress(espressoTEEVerifierAddress) {
+		log.Crit("Invalid EspressoTEEVerifierAddress provided")
+		return nil
```

Comment:
Wouldn't this still accept the zero address and lead to the return value of CallContract being inadvertently accepted?

Reply:
https://github.com/EspressoSystems/nitro-espresso-integration/pull/500/commits/51e8242f981ce5daaff6b609ddf51316ad17bc72